static/src/javascripts/projects/common/modules/ui/subscription-banners/subscription-banner.js

```diff
@@ -72,8 +72,8 @@ const hasAcknowledgedBanner = region =>
             return true;
         });
 
-const threeOrMorePageViews = (currentPageViews: number) =>
-    currentPageViews >= 3;
+const twoOrMorePageViews = (currentPageViews: number) =>
```

Comment:
does this change have any effect, given the `showFromFourthPageView` logic?

Reply:
`twoOrMorePageViews` is used when the banners are shown in the UK